lib/devise-security/models/password_archivable.rb

```diff
@@ -5,13 +5,13 @@
 
 module Devise
   module Models
-    # PasswordArchivable, this depends on the DatabaseAuthenticatable module from devise
```

Comment:
I think the existing is better as the module is still called `DatabaseAuthenticatable` even though the import statement looks different.